lib/models/Veeam.ts

```diff
@@ -26,9 +27,9 @@ export type VeeamSOSApiSchema = {
         LastModified?: string,
     },
     CapacityInfo?: {
-        Capacity: bigint | number,
-        Available: bigint | number,
-        Used: bigint | number,
+        Capacity: bigint | number | Long,
+        Available: bigint | number | Long,
+        Used: bigint | number | Long,
```

Comment:
why is it needed?
AFAIK we want to convert to/from Long inside MongoClientInterface : so the type handled throughout the code should not have `Long` anywhere...

Reply:
Actually this was done for the VeeamCapacityInfo serialize function here to be able to proceed , the thing is that we're storing this as a long in mongodb then expecting the variable to be typed as `bigint | number` on the function as it's taking the` capacity: VeeamSOSApiSchema['CapacityInfo']` as a param , which will not be the case as we're using the doc.value.x.x which is a long , this is why long was added in order to make the serialization possible. This will not affect the calculous as we're making the variables bigInts later on for calculous

Reply:
this however means that "user" of Capacity need (according to the compiler) to handle the `Long` type : which is exactly what we don't want, we want the `Long` type to stay in Mongo...

a better way seems to specify `bigint` here only (as it is what we want the code to deal with)
```suggestion
        Used: bigint,
```

and change the type of VeeamCapacityInfo.serialize() to be more permissive, like this:

```
static serialize(capacity: {
        Capacity: bigint | number | Long,
        Available: bigint | number | Long,
        Used: bigint | number | Long,
        LastModified?: string,
    }): VeeamSOSApiSerializable['CapacityInfo'] {
```